aws/sdk/integration-tests/s3/tests/connector-configuration.rs

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use aws_http::user_agent::AwsUserAgent;
+use aws_sdk_s3::{Client, Credentials, Region};
+use aws_smithy_client::test_connection::TestConnection;
+use aws_smithy_http::body::SdkBody;
+use aws_types::credentials::SharedCredentialsProvider;
+use aws_types::SdkConfig;
+use http::Uri;
+use std::convert::Infallible;
+use std::time::{Duration, UNIX_EPOCH};
+
+static INIT_LOGGER: std::sync::Once = std::sync::Once::new();
+
+fn init_logger() {
+    INIT_LOGGER.call_once(|| {
+        tracing_subscriber::fmt::init();
+    });
+}
+
+#[tokio::test]
+async fn test_http_connector_is_settable_in_config() {
```

Comment:
Is this test giving any benefit if the same thing is already exercised in the rest of the tests?

Reply:
No, not anymore. I'll delete it.